src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -840,17 +840,7 @@ var FormValidator = A.Component.create({
                     }
                 }
                 else if (!field.val()) {
-                    namedFieldNodes.each(
-                        function(node, index, nodeList) {
-                            node.removeClass('errorClass');
-                            node.removeAttribute('aria-invalid');
-                        }
-                    );
-
-                    if (fieldContainer) {
```

Comment:
From what I see in the code this part is not done in `resetField`. Are you sure this will keep working the same way? If so, can you explain?